packages/components/table/src/table.vue

```diff
@@ -294,12 +295,17 @@ export default defineComponent({
       return props.emptyText || t('el.table.emptyText')
     })
 
+    const columns = computed(() => {
+      return convertToRows(store.states.originColumns.value)[0]
+    })
+
```

Comment:
https://github.com/element-plus/element-plus/blob/ff936c76b56babae308383cc6909f20820dae961/packages/components/table/src/table-header/utils-helper.ts#L26

https://github.com/element-plus/element-plus/blob/ff936c76b56babae308383cc6909f20820dae961/packages/components/table/src/table-header/utils-helper.ts#L51C1-L54C4

`[0]` It is impossible to report an error.